kong/runloop/balancer.lua

```diff
@@ -209,6 +209,7 @@ do
   local balancer_types = {
     ["consistent"] = require("resty.dns.balancer.ring"),
```

Comment:
As discussed in https://github.com/Kong/kong/commit/3860151f2a2d9ef020d4c20496206aa5f922be9c#r33908240, let's also rename `consistent` to `consistent-hashing`